daffodil-cli/src/it/scala/org/apache/daffodil/cliTest/TestCLIUdfs.scala

```diff
@@ -179,105 +225,159 @@ class TestCLIUdfs {
-        " org.badudfs.evaluate.StringFunctions.FuncE." +
-        " Unsupported return type: String[]")
+      cli.expectErr(
+        "[warn] User Defined Function ignored:" +
+          " org.badudfs.evaluate.StringFunctions.Replace." +
+          " Missing evaluate method: urn:example:com:ext:badudfs:stringfunctions:replace",
+      )
+      cli.expectErr(
+        "[warn] User Defined Function ignored:" +
+          " org.badudfs.evaluate.StringFunctions.FuncA." +
+          " Overloaded evaluate method: urn:example:com:ext:badudfs:stringfunctions:funcA",
+      )
+      cli.expectErr(
+        "[warn] User Defined Function ignored:" +
+          " org.badudfs.evaluate.StringFunctions.FuncB." +
+          " Unsupported return type: void",
+      )
+      cli.expectErr(
+        "[warn] User Defined Function ignored:" +
+          " org.badudfs.evaluate.StringFunctions.FuncC." +
+          " Unsupported parameter type(s): String[],int[]",
+      )
+      cli.expectErr(
+        "[warn] User Defined Function ignored:" +
+          " org.badudfs.evaluate.StringFunctions.FuncD." +
+          " Unsupported parameter type(s): String[]",
+      )
+      cli.expectErr(
+        "[warn] User Defined Function ignored:" +
+          " org.badudfs.evaluate.StringFunctions.FuncE." +
+          " Unsupported return type: String[]",
+      )
       cli.expectErr("[error] Schema Definition Error: Unsupported function: jsudf:replace")
-    } (ExitCode.UnableToCreateProcessor)
+    }(ExitCode.UnableToCreateProcessor)
   }
 
   /**
    * Tests the case when a function class:
    *   throws custom error on evaluate
    */
   @Test def test_UDFClass_CustomExceptionOnEvaluate(): Unit = {
-    val schema = path("daffodil-udf/src/test/resources/org/apache/daffodil/udf/genericUdfSchema.xsd")
-    val classpath = udfClasspath("daffodil-udf/src/test/scala/org/sbadudfs/udfexceptions/evaluating/StringFunctions/")
+    val schema = path(
+      "daffodil-udf/src/test/resources/org/apache/daffodil/udf/genericUdfSchema.xsd",
+    )
+    val classpath = udfClasspath(
+      "daffodil-udf/src/test/scala/org/sbadudfs/udfexceptions/evaluating/StringFunctions/",
+    )
 
     runCLI(args"parse -s $schema -r user_func2", classpath) { cli =>
       cli.send("strng", inputDone = true)
-      cli.expectErr("[error] User Defined Function 'ssudf:reverse' Error. Cause: org.sbadudfs.udfexceptions.evaluating.StringFunctions.Reverse$CustomException: UDF Error!")
-      cli.expectErr("at org.sbadudfs.udfexceptions.evaluating.StringFunctions.Reverse.evaluate(StringFunctionsProvider.scala:56)")
-    } (ExitCode.UserDefinedFunctionError)
+      cli.expectErr(
+        "[error] User Defined Function 'ssudf:reverse' Error. Cause: org.sbadudfs.udfexceptions.evaluating.StringFunctions.Reverse$CustomException: UDF Error!",
+      )
+      cli.expectErr(
+        "at org.sbadudfs.udfexceptions.evaluating.StringFunctions.Reverse.evaluate(StringFunctionsProvider.scala:56)",
```

Comment:
Looks like this test is failing due to line numbers change from formatting. We should remove the line number from this tests. Our tests don't need to be that specific.